capa/features/extractors/strings.py

```diff
@@ -11,11 +11,12 @@
 import re
 from collections import namedtuple
 
+DEFAULT_STRING_LENGTH = 4
 ASCII_BYTE = r" !\"#\$%&\'\(\)\*\+,-\./0123456789:;<=>\?@ABCDEFGHIJKLMNOPQRSTUVWXYZ\[\]\^_`abcdefghijklmnopqrstuvwxyz\{\|\}\\\~\t".encode(
     "ascii"
 )
-ASCII_RE_4 = re.compile(b"([%s]{%d,})" % (ASCII_BYTE, 4))
-UNICODE_RE_4 = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE, 4))
+ASCII_RE_4 = re.compile(b"([%s]{%d,})" % (ASCII_BYTE, DEFAULT_STRING_LENGTH))
+UNICODE_RE_4 = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE, DEFAULT_STRING_LENGTH))
```

Comment:
```suggestion
ASCII_RE_DEFAULT = re.compile(b"([%s]{%d,})" % (ASCII_BYTE, DEFAULT_STRING_LENGTH))
UNICODE_RE_DEFAULT = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE, DEFAULT_STRING_LENGTH))
```

since the length of the string is now derived from the constant, lets change the name.